qiskit/transpiler/passmanager.py

```diff
@@ -31,6 +31,7 @@
 from .exceptions import TranspilerError
 from .layout import TranspileLayout
 
+# pylint: disable=invalid-name
 _CircuitsT = Union[List[QuantumCircuit], QuantumCircuit]
```

Comment:
I don't actually think this is a false positive, I think it's indicative of the typing construct being wrong.  This defines an alias, but the way it's used is actually intended as a typevar generic like:
```python
from typing import TypeVar, Union, List

_CircuitsT = TypeVar("_CircuitsT", bound=Union[List[QuantumCircuit], QuantumCircuit])
```

Reply:
Well it was complaining about the name, not the type. You had to infer the name didn't match the rules for type aliases and figure out it's because it wasn't using the right thing here. But yeah I'll update to use this instead.

Reply:
judging by my local branch that did this and half-remembered confusion, I think I went down the same rabbit hole as you a couple of months ago yeah